CollapseLauncher/CollapseLauncher.csproj

```diff
@@ -32,67 +41,73 @@
 		- ENABLEHTTPREPAIR : Override HTTPS download scheme to HTTP while downloading repair files
 		- SIMULATEAPPLYPRELOAD : Simulates downloading and applying preload update, regardless of current working game version (WILL BREAK GAME)
 		- DUMPGIJSON : Dumps Genshin Impact GeneralData JSON registry into console (CPU INTENSIVE LOAD!)
-    - SIMULATEGIHDR : Force enable Genshin Impact HDR settings panel for non-HDR panels (Note: this wont force GI to use HDR)
+		- SIMULATEGIHDR : Force enable Genshin Impact HDR settings panel for non-HDR panels (Note: this wont force GI to use HDR)
+		- GSPBYPASSGAMERUNNING : Bypass checks for Game Settings Pages for currently running game
+		- MHYPLUGINSUPPORT : Enable miHoYo's patching system through launcher (EXPERIMENTAL)
```

Comment:
Can we elaborate on what this does exactly?

Reply:
This is for launcher patching support where they sometime push an "optional" patch (though sometime its quite critical like HDR bug on HSR recently) through the plugin API (similar to resource API). This has quite complicated implementation because the API doesn't state which version is the served patch for. So it remains experimental for now